### ISSUE TITLE
Is there any plan to update magic_enum?

### ISSUE THREAD

**Issue by toge** [teslamotors/fixed-containers#101]
magic_enum has been changed include path since 0.9.4.
https://github.com/Neargye/magic_enum/releases/tag/v0.9.4

Is there any plan to update magic_enum?

**Comment by alexkaratarakis** [comment]
https://github.com/teslamotors/fixed-containers/pull/104 should do it.
Note that when adding `fixed_containers` to your workspace, you can override the version with your own using `repo_mapping`, for example:
```
http_archive(
    name = "fixed_containers",
    urls = ["https://artifactory.teslamotors.com/artifactory/ap-github-proxy/teslamotors/fixed-containers/archive/eb7764c4c5057c696522fd48474160921cfb2a69.tar.gz"],
    strip_prefix = "fixed-containers-eb7764c4c5057c696522fd48474160921cfb2a69",
    sha256 = "e552247252155d49e1550d2387bf984dffd9d1e7f3447fd91a194b7b36f77eb9",
    repo_mapping = {"@com_github_neargye_magic_enum" : "@your_magic_enum_target"},
)
```

**Comment by toge** [comment]
@alexkaratarakis 
Thanks a lot!
